src/mono/wasm/host/DevServer/DevServerStartup.cs

```diff
@@ -57,7 +57,9 @@ public static void Configure(IApplicationBuilder app, IOptions<DevServerOptions>
             });
         }
 
-        app.UseBlazorFrameworkFiles();
+        //app.UseBlazorFrameworkFiles();
```

Comment:
leaving in a form of a comment?

Reply:
I'm copying changes from Blazor. They have it commented out. I left it the same for better code matching...